libs/transactions/src/lib/views/transactions-view/transactions-view.component.spec.ts

```diff
@@ -91,11 +89,26 @@ describe('TransactionsViewComponent', () => {
   };
 
   it('should set title from the route data', () => {
+    const snapshot = {
```

Comment:
This variable can be moved to the upper level or to the `beforeEach` block to avoid duplicating.

Reply:
I did some refactoring. Please take another look